clients/src/main/java/org/apache/kafka/clients/consumer/internals/RequestManagers.java

```diff
@@ -32,7 +32,7 @@ public class RequestManagers {
 
     public final Optional<CoordinatorRequestManager> coordinatorRequestManager;
     public final Optional<CommitRequestManager> commitRequestManager;
-    private final Optional<HeartbeatRequestManager> heartbeatRequestManager;
+    public final Optional<HeartbeatRequestManager> heartbeatRequestManager;
```

Comment:
I will revert this change since we didn't end up using the heart beat request manager elsewhere.